src/analyses/ai.cpp

```diff
@@ -79,18 +54,9 @@ void ai_baset::output(
   }
 }
 
-/*******************************************************************\
-
-Function: ai_baset::output_json
-
-  Inputs: The namespace and goto_functions
-
- Outputs: The JSON object
-
- Purpose: Output the domains for the whole program as JSON
-
-\*******************************************************************/
-
+/// Output the domains for the whole program as JSON
+/// parameters: The namespace and goto_functions
```

Comment:
This does not render correctly in the doxygen output. It's just concatenated to the previous line.